src/context/WalletProvider/WalletProvider.tsx

```diff
@@ -651,6 +651,11 @@ export const WalletProvider = ({ children }: { children: React.ReactNode }): JSX
               /** List of RPC URLs indexed by chain ID */
               rpc: {
                 1: getConfig().REACT_APP_ETHEREUM_NODE_URL,
+                10: getConfig().REACT_APP_OPTIMISM_NODE_URL,
+                56: getConfig().REACT_APP_BNBSMARTCHAIN_NODE_URL,
+                100: getConfig().REACT_APP_GNOSIS_NODE_URL,
+                137: getConfig().REACT_APP_POLYGON_NODE_URL,
+                43114: getConfig().REACT_APP_AVALANCHE_NODE_URL,
```

Comment:
```suggestion
                [CHAIN_REFERENCE.EthereumMainnet]: getConfig().REACT_APP_ETHEREUM_NODE_URL,
                [CHAIN_REFERENCE.Optimism]: getConfig().REACT_APP_OPTIMISM_NODE_URL,
                [CHAIN_REFERENCE.BnbSmartChainMainnet]: getConfig().REACT_APP_BNBSMARTCHAIN_NODE_URL,
                [CHAIN_REFERENCE.GnosisMainnet]: getConfig().REACT_APP_GNOSIS_NODE_URL,
                [CHAIN_REFERENCE.PolygonMainnet]: getConfig().REACT_APP_POLYGON_NODE_URL,
                [CHAIN_REFERENCE.AvalancheCChain]: getConfig().REACT_APP_AVALANCHE_NODE_URL,
```

Reply:
100%. Addressed in https://github.com/shapeshift/web/pull/4590/commits/b6c2b83bf461c9ffc074882fe7a79a8f8d98e320.